README.md

```diff
@@ -62,61 +65,63 @@ Finally, two sample .NET apps interact with the resources deployed to showcase t
-  * Load the variables by running:
+   2. The newly created `.sh` file should now open in Visual Studio Code.
+   3. Edit the values to your preference, ensure `TENANT_ID` corresponds to your Azure tenant.
+   4. For `PREFIX` use a short 5 character string that can be unique for the resource name composition.
+   5. Load the variables by running:
+
+    ```bash
+    source ./temp/envvars.sh
+    ```
 
-  ```bash
-  source ./temp/envvars.sh
-  ```
+#### Azure Resources with Virtual Network, private Key Vault, private Storage and Application Gateway
 
-#### Azure Resources with Virtual Network, private Storage and Application Gateway
+Deploy the Azure components for setting up Virtual Network, self-signed SSL certificate, Storage, Private Link, custom private DNS and Application Gateway configured to talk to the Storage account. This allows you to validate the flow before configuring Azure IoT Hub and client communication. The SSL certificate is stored in Azure Key Vault.
 
-Deploy the Azure components for setting up Virtual Network, self-signed SSL certificate, Storage, Private Link, custom private DNS and Application Gateway configured to talk to the Storage account. This allows you to validate the flow before configuring Azure IoT Hub and client communication.
+> [! WARNING]
+> [Azure Key Vault in this sample has network rules to deny public internet access, but has an exception to allow the current user's IP address for this quickstart setup. In real environments we recommend removing this rule and integrating your service access through [Private Endpoint](https://learn.microsoft.com/en-us/azure/key-vault/general/private-link-service?tabs=portal).]
 
-* From the root directory of this repo, run the first part of the deployment. The script will use the environment variables and build composed resource names by appending the `PREFIX` variable as Azure resource names.
+1. From the root directory of this repo, run the first part of the deployment. The script will use the environment variables and build composed resource names by appending the `PREFIX` variable as Azure resource names.
 
 ```bash
 ./deploy/quickstart-agw-storage.sh
 ```
 
-* It will take a few minutes to deploy all resources. Keep the terminal open and take note of some of the generated DNS entries.
-* This scripts also uploads a `sample.txt` file to a Blob storage container and generates a SAS URI token for testing.
-* From a bash terminal, try out a `CURL` command to the DNS of the Public IP address attached to the Application Gateway.
+2. It will take a few minutes to deploy all resources. Keep the terminal open and take note of some of the generated DNS entries.
+3. This scripts also uploads a `sample.txt` file to a Blob storage container and generates a SAS URI token for testing.
+4. From a bash terminal, try out a `CURL` command to the DNS of the Public IP address attached to the Application Gateway.
```

Comment:
```suggestion
   It will take a few minutes to deploy all resources. Keep the terminal open and take note of some of the generated DNS entries.

   This scripts also uploads a `sample.txt` file to a Blob storage container and generates a SAS URI token for testing.

1. From a bash terminal, try out a `CURL` command to the DNS of the Public IP address attached to the Application Gateway.
```